### PR TITLE
common: set env var I_AM_IN_A_CONTAINER

### PR DESCRIPTION
**Pull request by guits**
So any tool can detect whether it is actually running
from within a container.

Signed-off-by: Guillaume Abrioux <gabrioux@redhat.com>

### CODE REVIEW

**Comment by guits** [comment]
@leseb FYI

**Comment by sebastian-philipp** [comment]
relates to 

https://github.com/ceph/ceph/blob/a67d1cf2a7a4031609a5d37baa01ffdfef80e993/src/common/hostname.cc#L24

https://github.com/ceph/ceph/blob/c65798342a5bcc66b8c3c3d96132fc68d893843a/src/msg/Messenger.cc#L24

**Comment by guits** [comment]
jenkins test ceph_ansible-lvm_osds

File: tests/setup.yml [hunk 1]
```diff
@@ -48,9 +48,14 @@
     - name: allow nodes to use an insecure registry - podman
       ini_file:
         path: /etc/containers/registries.conf
-        section: registries.insecure
-        option: registries
-        value: "['{{ ceph_docker_registry }}']"
+        section: "[registry]"
+        option: "{{ item.key }}"
+        value: "{{ item.value }}"
+      loop: "{{ insecure_registry_config | dict2item }}"
```
Comment (dsavineau) [inline tests/setup.yml:54]:
```suggestion
      loop: "{{ insecure_registry_config | dict2items }}"
```

**Review by dsavineau — CHANGES_REQUESTED** [review]

**Review by leseb — APPROVED** [review]

**Pull request merged**